shared/middlewares/security.js

```diff
@@ -2,6 +2,9 @@
 const hpp = require('hpp');
 const helmet = require('helmet');
 const uuid = require('uuid');
+const hsts = require('hsts');
+const express_enforces_ssl = require('express-enforces-ssl');
+const IS_PROD = process.env.NODE_ENV === 'production';
```

Comment:
```suggestion
const IS_PROD = process.env.NODE_ENV === 'production' && !process.env.FORCE_DEV;
```

That will fix tests :+1: